Objects/codeobject.c

```diff
@@ -1906,3 +1906,18 @@ _PyCode_ConstantKey(PyObject *op)
     }
     return key;
 }
+
+
+void 
+_PyStaticCode_Dealloc(PyCodeObject *co, _Py_CODEUNIT *firstinstr) {
```

Comment:
The curly braces of a function body belong on the next line, https://www.python.org/dev/peps/pep-0007/#code-lay-out

Reply:
Thanks added